LICENSE-MPL-RabbitMQ

```diff
@@ -327,7 +327,7 @@
      shall survive.
 
      8.2.  If You initiate litigation by asserting a patent infringement
-     claim (excluding declatory judgment actions) against Initial Developer
+     claim (excluding declaratory judgment actions) against Initial Developer
```

Comment:
As noted above and in https://github.com/rabbitmq/rabbitmq-common/pull/302#discussion_r255809184 this change should probably be subject to approval by a lawyer)

Reply:
For that reason, please back it out. We won't be changing any license files.